### PR TITLE
Mybranch

### PR DESCRIPTION
**Pull request by Yantiomene**
file rectangle.py just commited and push

### CODE REVIEW

File: rectangle.py [hunk 1]
```diff
@@ -0,0 +1,54 @@
+from math import sqrt
+
+#this function find if parameters are for a same rectange
```
Comment (eFunzo) [inline rectangle.py:3]: This comment is not descriptive enough.

**Review by eFunzo — APPROVED** [review]

**Review by kontchou-rovil — APPROVED** [review]

**Comment by eFunzo** [comment]
main comment.

File: rectangle.py [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+def is_rectangle(a, b, c, d, e):
+    dico = {}
+    
+    liste = [a, b, c, d, e]
+    liste.sort() # we sort the list to find smalest values
+    
+    #here we find the width and the length depending on the dimensions
+    if liste[0] >= 1:
+        dico["width"] = liste[0]
+        dico["length"] = liste[1]
```
Comment (eFunzo) [inline rectangle.py:13]: just a comment at line 13.

File: rectangle.py [hunk 3]
```diff
@@ -0,0 +1,54 @@
+from math import sqrt
+
+#this function find if parameters are for a same rectange
+def is_rectangle(a, b, c, d, e):
+    dico = {}
+
```
Comment (eFunzo) [inline rectangle.py:6]: starting review

File: rectangle.py [hunk 4]
```diff
@@ -0,0 +1,54 @@
+from math import sqrt
+
+#this function find if parameters are for a same rectange
+def is_rectangle(a, b, c, d, e):
+    dico = {}
+    
+    liste = [a, b, c, d, e]
+    liste.sort() # we sort the list to find smalest values
```
Comment (kontchou-rovil) [inline rectangle.py:8]: this line is clear

**Review by eFunzo — CHANGES_REQUESTED** [review]

**Pull request merged**